gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -21,22 +22,13 @@ class <%= service.client_name %>
   # @private
   attr_reader :<%= service.stub_name %>
 
-  # The default address of the service.
-  SERVICE_ADDRESS = "<%= service.client_address %>"
```

Comment:
Replacing these constants with the configuration object is the big win IMO.

Reply:
Yes, especially because we explicitly need these to be globally customizable moving forward.